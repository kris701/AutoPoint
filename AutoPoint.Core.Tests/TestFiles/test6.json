{
  "Includes": [
    "test4.json"
  ],
  "Branch": {
    "name": "simple4",
    "Nodes": [
      {
        "name": "Subnode",
        "Nodes": [
          {
            "Name": "test2",
            "Route": "abc"
          }
        ]
      }
    ]
  }
}
