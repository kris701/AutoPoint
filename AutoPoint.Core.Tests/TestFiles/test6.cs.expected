// This document is auto generated!
public static class simple4 {
	public const string Name = "simple4";
	public static class Subnode {
		public const string Name = "simple4/subnode";
		public const string test2 = "simple4/subnode/abc";
		public const string test = "simple4/subnode";
	}
	public const string test = "simple4/abc";
	public const string test2 = "simple4/123";
}
